qcodes/instrument_drivers/Keysight/N5222B.py

```diff
@@ -11,6 +11,7 @@ def __init__(self, name: str, address: str, **kwargs: Any):
                          nports=4,
                          **kwargs)
 
-        options = self.get_options()
-        if "419" in options:
-            self._set_power_limits(min_power=-90, max_power=13)
+        attenuators_options = {'217', '219', '220', '417', '419', '420'}
+        options = set(self.get_options())
+        if attenuators_options.intersection(options):
+            self._set_power_limits(min_power=-95, max_power=13)
```

Comment:
Is is correct that all these options results in the same power limits ?

Reply:
Hi Jens,

Yes, I think so. Please see the following links for options on the various PNA models:

https://na.support.keysight.com/vna/help/latest/Support/Configurations.htm#N522x
https://na.support.keysight.com/pna/help/latest/Support/Configurations.htm

It states that the attenuation is the same for these options, but varies for every PNA model.